django_x509/tests/test_ca.py

```diff
@@ -614,3 +614,11 @@ def test_generate_ca_with_passphrase(self):
         ca.full_clean()
         ca.save()
         self.assertIsInstance(ca.pkey, crypto.PKey)
+
+    def test_datetime_to_string(self):
+        generalized_datetime = datetime(2050, 1, 1, 0, 0, 0, 0)
+        utc_datetime = datetime(2049, 12, 31, 0, 0, 0, 0)
+        self.assertEqual(datetime_to_string(generalized_datetime),
+                         generalized_datetime.strftime(generalized_time))
+        self.assertEqual(datetime_to_string(utc_datetime),
+                         utc_datetime.strftime(utc_time))
```

Comment:
I did not use
```
def test_generate_ca_with_generalizedTime(self):
        """ Any date after 2049, should be in generalizedTime"""
        ca = self._create_ca(validity_start=datetime(2050, 1, 1, 0, 0, 0, 0),
                             validity_end=datetime(2070, 1, 1, 0, 0, 0, 0))
        ca.full_clean()
        ca.save()
        self.assertEqual(datetime_to_string(ca.validity_start),
                         ca.validity_start.strftime(generalized_time))
        self.assertEqual(datetime_to_string(ca.validity_end),
                         ca.validity_end.strftime(generalized_time))
```
Because it basically checks datetime object and Ca object has no significant role in this check.

Reply:
better, thanks